dashboard/settings.py

```diff
@@ -330,20 +330,7 @@
 def show_debug_toolbar(request):
     return DEBUG and request.user and request.user.is_authenticated and request.user.is_superuser
 
-DEBUG_TOOLBAR_PANELS = [
-    'debug_toolbar.panels.versions.VersionsPanel',
-    'debug_toolbar.panels.timer.TimerPanel',
-    'debug_toolbar.panels.settings.SettingsPanel',
-    'debug_toolbar.panels.headers.HeadersPanel',
-    'debug_toolbar.panels.request.RequestPanel',
-    'debug_toolbar.panels.sql.SQLPanel',
-    'debug_toolbar.panels.staticfiles.StaticFilesPanel',
-    'debug_toolbar.panels.templates.TemplatesPanel',
-    'debug_toolbar.panels.cache.CachePanel',
-    'debug_toolbar.panels.signals.SignalsPanel',
-    'debug_toolbar.panels.logging.LoggingPanel',
-    'debug_toolbar.panels.redirects.RedirectsPanel',
-]
+DEBUG_TOOLBAR_PANELS = dt_settings.PANELS_DEFAULTS
```

Comment:
Could not find PANELS_DEFAULTS defined in dt_settings.py

Reply:
Is this an error you see what you run this? It should be there https://github.com/jazzband/django-debug-toolbar/blob/master/debug_toolbar/settings.py and I'd thought it was working. I'll can look at this again.